t/05-fullstack-scalability.t

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env perl -w
+
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+use strict;
+use warnings;
+
+use Mojo::File qw(path tempdir);
+
+my $tempdir;
+BEGIN {
+    use FindBin;
+    $tempdir             = tempdir;
+    $ENV{OPENQA_BASEDIR} = $tempdir->child('t', 'scheduler');
+    $ENV{OPENQA_CONFIG}  = path($ENV{OPENQA_BASEDIR}, 'config')->make_path;
+    my ($data_dir, $config_dir) = (path($FindBin::Bin, 'data'), path($ENV{OPENQA_CONFIG}));
+    $data_dir->child($_)->copy_to($config_dir->child($_)) for qw(openqa.ini database.ini workers.ini);
+    path($ENV{OPENQA_BASEDIR}, 'openqa', 'db')->make_path->child('db.lock')->spurt;
```

Comment:
I guess this is copy pasted and needs to be extracted? Do we actually still need the DB file parts?